repos/system_upgrade/el8toel9/actors/firewalldcheckservicetftpclient/actor.py

```diff
@@ -0,0 +1,35 @@
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.models import FirewalldUsedObjectNames
+from leapp.reporting import create_report, Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class FirewalldCheckServiceTftpClient(Actor):
+    '''
+    No documentation has been provided for the firewalld_check_service_tftp_client actor.
+    '''
+
+    name = 'firewalld_check_service_tftp_client'
+    consumes = (FirewalldUsedObjectNames,)
+    produces = (Report,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        tftp_client_service = 'tftp-client'
+        send_report = False
+
+        for facts in self.consume(FirewalldUsedObjectNames):
+            if tftp_client_service in facts.services:
+                send_report = True
+
+        if send_report:
+            create_report([
+                reporting.Title('Unsupported Firewalld Configuration'),
+                reporting.Summary('Firewalld has enabled service '
```

Comment:
Sorry for nitpicking, imho 'Firewalld has service {service} enabled..' reads better here

Reply:
Done